acbuild/acbuild.go

```diff
@@ -61,30 +61,22 @@ var (
 	cmdExitCode int
 )
 
-func tmpacipath() string {
-	return path.Join(contextpath, workprefix, "currentaci")
-}
-
-func targetpath() string {
-	return path.Join(contextpath, workprefix, "target")
-}
-
-func scratchpath() string {
-	return path.Join(contextpath, workprefix, "scratch")
-}
-
-func depstorepath() string {
-	return path.Join(contextpath, workprefix, "depstore")
-}
-
-func workpath() string {
-	return path.Join(contextpath, workprefix, "work")
-}
-
 func lockpath() string {
 	return path.Join(contextpath, workprefix, "lock")
 }
 
+func newACBuild() *lib.ACBuild {
```

Comment:
Can you pass contextpath and workprefix to newACBuild? global vars always make me uneasy, especially when you are mutating them at times outside of flag parsing.